docs/src/UserGuide/faq.md

```diff
@@ -77,46 +77,224 @@ Now we can concatenate `ds1` and `ds2`:
 dsfinal = concatenatecubes([ds1, ds2], Dim{:Variables}(["var1", "var2"]))
 ````
 
-## How do I subset a Cube?
+## How do I subset a YAXArray, Cube or Dataset?
 
-Let's start by creating a dummy cube. Define the time span of the cube
+These are the three main datatypes provided by the YAXArrays libray. You can find a description of them [here](https://juliadatacubes.github.io/YAXArrays.jl/dev/UserGuide/types).
+
+### Subsetting a YAXArray
+
+Let's start by creating a dummy YAXArray.
+
+Firstly, load the required libraries
+````@example howdoi
+using YAXArrays
+usinng Dates # To generate the dates of the time axis
+using DimensionalData # To use the "Between" option for selecting data
+````
+
+Define the time span of the YAXArray
 
 ````@example howdoi
-using Dates
 t = Date("2020-01-01"):Month(1):Date("2022-12-31")
 ````
 
-create cube axes
+create YAXArray axes
 
 ````@example howdoi
 axes = (Dim{:Lon}(1:10), Dim{:Lat}(1:10), Dim{:Time}(t))
 ````
 
-assign values to a cube
+create the YAXArray
+
+````@ansi howdoi
+y = YAXArray(axes, reshape(1:3600, (10, 10, 36)))
+````
+````@ansi howdoi
```

Comment:
all the second ansi blocks are not necessary. That is the output from the previous one, after execution.

Reply:
But I separate them in order for the  user to easily copy the input and paste and run it in the console in a quick way. If both blocks were the same, it would be not so ease. The button on the up-right corner of the block, would copy both the input and the output.

Reply:
you can always see a copy button in the output cells: 
<img width="931" alt="Screenshot 2024-07-22 at 17 04 52" src="https://github.com/user-attachments/assets/af25ea02-99f2-415f-8963-320985dd7e4e">

> be different for the input than for the output?.

Yes, we could try different styles.

Reply:
I can achieve this behavior? Could you suggest me a link to a documentation page for this?

Reply:
I have just seen how to do it.